### PR TITLE
16.0 fix nightly las

### PR DESCRIPTION
**Pull request by smetl**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/spreadsheet_account/tests/test_account_group.py [hunk 1]
```diff
@@ -1,80 +1,41 @@
...
         self.assertEqual(self.env["account.account"].get_account_group([]), [])
 
     def test_fetch_account_one_group(self):
-        account_ids = self.env["account.account"].search([("account_type", "=", "income_other"), ("company_id", "=", self.env.company.id), ("code", "!=", "456")])
-        self._unlink_default_accounts(account_ids)
-        self.assertEqual(
-            self.env["account.account"].get_account_group(["income_other"]),
-            [["456"]],
-        )
+        self.assertEqual(self.env["account.account"].get_account_group(['income_other']), [['450000']])
```
Comment (vin-odoo) [inline addons/spreadsheet_account/tests/test_account_group.py:12]: In the test with multiple accounts, you first unlink the accounts and then create two new ones.
But in this one, you use the already existing one, why the difference?

We can argue with this, but isn't it better to create a new one here too like it was done before to be 100% deterministic?

**Review by vin-odoo — CHANGES_REQUESTED** [review]
Not that much to say

**Comment by vin-odoo** [comment]
@robodoo r+ rebase-ff

Good job!

File: addons/account/tests/test_tour.py [hunk 2]
```diff
@@ -25,4 +26,11 @@ def test_01_account_tour(self):
         all_moves = self.env['account.move'].search([('move_type', '!=', 'entry')])
         all_moves.button_draft()
         all_moves.with_context(force_delete=True).unlink()
+
+        # In case of latam impacting multiple countries, disable the required fields manually.
+        if 'l10n_latam_use_documents' in self.env['account.journal']._fields:
+            self.env['account.journal']\
+                .search([('company_id', '=', self.env.company.id), ('type', '=', 'sale')])\
+                .write({'l10n_latam_use_documents': False})
```
Comment (william-andre) [inline addons/account/tests/test_tour.py:34]: Once again, this is just hiding stuff.
This tour needs to work for users, it is the onboarding tour.

Comment (smetl) [reply addons/account/tests/test_tour.py:34]: For now, I followed the logic of the setup disabling the l10n stuff to make it running on runbot.

**Pull request closed without merging**